### DISCUSSION TITLE
How to use hyde theme import

### DISCUSSION THREAD

**Discussion by sitolam** [prasanthrangan/hyprdots#1960] [Q&A] [answered]
If I type `Hyde theme import` nothing happens but when I try `Hyde theme patch` it doesn't show any themes:
![image](https://github.com/user-attachments/assets/05b7b300-8640-4197-b558-f7a6c8754012)

**Comment by kRHYME7** [comment]
updated the CLI already?
Can you ping github?

**Reply by sitolam** [reply]
1. How do I update the CLI? I'm now on version: v0.5.2-119-g8646cff
2. Yes
This message also pops up when I leave the menu:
`sed: can't read /home/<username>/.cache/hyde/landing/hyde-gallery/hyde-themes.db: No such file or directory`

**Reply by kRHYME7** [reply, marked as answer]
run `Hyde update ` 

or ` yay -S hyde-cli-git ` 

we are on` Version: v0.6.0-14-gc8e4e85 ` 
![image](https://github.com/user-attachments/assets/64c36f48-5434-484e-a9f6-ace547d95605)

**Reply by kRHYME7** [reply]
Outdated cli. We are using json now for the theme database

**Reply by sitolam** [reply]
> run `Hyde update ` 
> 
> or ` yay -S hyde-cli-git ` 
> 
> we are on` Version: v0.6.0-14-gc8e4e85 ` 
> ![image](https://github.com/user-attachments/assets/64c36f48-5434-484e-a9f6-ace547d95605)
> 

`yay -S hyde-cli-git` fixed it
